system/lib/libc/musl/src/thread/pthread_cancel.c

```diff
@@ -14,6 +14,7 @@ long __cancel()
 	return -ECANCELED;
 }
 
+#ifndef __EMSCRIPTEN__
```

Comment:
I think can move this up to line 5, no?

Reply:
Yep, fixed with commit f4058e2e8aa4fdda6aed317327d654b898371c2a.